src/content/docs/reference/builders.mdx

```diff
@@ -7,26 +7,35 @@ sidebar:
 import Aside from '@components/Aside.astro'
 
 Builders are responsible for creating a container image with everything you need to be productive.
-Daytona allows for control over how the resulting Project environment is created, allowing you to choose your preferred method to pull in your project's tools, dependencies, and configuration.
+Daytona provides control over how the resulting Project environment is created, allowing you to choose your preferred method to pull in your project's tools, dependencies, and configuration.
 
-At Workspace creation time, you can choose between the following options:
+When creating a Workspace, you can choose between the following Builders during the Advanced Configuration step:
 
-1.  __Automatic__  
+1.  [__Automatic__](#automatic)  
     Daytona builds the Project image after auto-detecting the most appropriate Builder.
 2.  [__Dev Container__](#dev-container)  
     Daytona builds the Project image according to the Dev Container configuration located in the project.
 3.  [__Custom Image__](#custom-image)  
     Daytona builds the Project image using an image you specify.
-4.  __None__  
-    Daytona creates the Project using the default base image (`daytonaio/workspace-project`).
+4.  [__None__](#none)  
+    Daytona creates the Project using the default base image.
 
 ## Supported Builders
+### Automatic
+Daytona can auto-detect the most appropriate Builder for your project.
+When choosing this option, Daytona uses the following logic to determine which Builder to use:
+
+1.  Check if a Dev Container configuration is present at `.devcontainer/devcontainer.json`, relative to the repository root.
```

Comment:
We check `.devcontainer.json` and `.devcontainer/devcontainer.json` as per the devcontainer spec.

Reply:
Updated in [48e8a9a](https://github.com/daytonaio/docs/pull/56/commits/48e8a9ae70cfccac2eed1bd0409a9395a76a64ca)